Bio/Seq.py

```diff
@@ -947,7 +947,9 @@ def transcribe(self):
         >>> my_protein.transcribe()
         Seq('MAIVMGRU')
         """
-        return Seq(str(self).replace("T", "U").replace("t", "u"))
+        return Seq(
+            str.translate(str(self), {84: 85, 116: 117})
+        )  # str.maketrans("Tt", "Uu")
```

Comment:
The current implementation using `bytes` is about 12 times faster than `str.translate`.